keyboards/claw44/keymaps/default/keymap.c

```diff
@@ -0,0 +1,174 @@
+     KC_TAB , KC_A   , KC_S    , KC_D   , KC_F    , KC_G   ,     KC_H   , KC_J    , KC_K   , KC_L    , KC_SCLN, KC_QUOT,
+  //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+     KC_LSFT, KC_Z   , KC_X    , KC_C   , KC_V    , KC_B   ,     KC_N   , KC_M    , KC_COMM, KC_DOT  , KC_SLSH, KC_RSFT,
+  //`--------+--------+---------+--------+---------+--------/   \--------+---------+--------+---------+--------+--------'
+                       KC_A_DEL, KC_G_EN, KC_L_SPC, KC_C_BS,     KC_C_BS, KC_R_ENT, KC_G_JA, KC_A_DEL
+  //                 `----------+--------+---------+--------'   `--------+---------+--------+---------'
+  ),
+
+  //   \ ^ ! & |  @ = + * % -
+  // ( # $ " ' ~  ← ↓ ↑ → ` )
+  //         { [  ] }
+
+  [_RAISE] = LAYOUT( \
+  //,--------+--------+--------+--------+--------+--------.   ,--------+--------+--------+--------+--------+--------.
+     _______, KC_BSLS, KC_CIRC, KC_EXLM, KC_AMPR, KC_PIPE,     KC_AT  , KC_EQL , KC_PLUS, KC_ASTR, KC_PERC, KC_MINS,
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+     KC_LPRN, KC_HASH, KC_DLR , KC_DQT , KC_QUOT, KC_TILD,     KC_LEFT, KC_DOWN,  KC_UP , KC_RGHT, KC_GRV , KC_RPRN,
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+     _______, _______, _______, _______, KC_LCBR, KC_LBRC,     KC_RBRC, KC_RCBR, _______, _______, _______, _______,
+  //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                       _______, _______, _______, _______,     _______, _______, _______, RESET
+  //                  `--------+--------+--------+--------'   `--------+--------+--------+--------'
+  ),
+
+  [_LOWER] = LAYOUT( \
+  //,--------+--------+--------+--------+--------+--------.   ,--------+--------+--------+--------+--------+--------.
+     KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  ,     _______, KC_EQL , KC_PLUS, KC_ASTR, KC_PERC, KC_MINS,
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+     _______, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,     KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , _______,
+  //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+     KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,     _______, _______, KC_COMM, KC_DOT , KC_SLSH, _______,
+  //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                       RESET  , _______, _______, _______,     _______, _______, _______, _______
+  //                  `--------+--------+--------+--------'   `--------+--------+--------+--------'
+  ),
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
I removed that function.